geocoder_tester/base.py

```diff
@@ -108,6 +110,8 @@ def _common_params(self, **kwargs):
             params['accept-language'] = kwargs['lang']
         return params
 
+    def _transform_search_results(self, results):
+        return results
```

Comment:
Just noticed that the Photon flavour does not work anymore while testing. This function needs to go into the `GenericApi` class, of course.

Reply:
Oh yes, indeed. Moved it up.